src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1532,6 +1532,16 @@ Notifications will only be sent for jobs whose generic information contains user
 </genericInformation>
 ----
 
+=== Chaining Jobs
+
+The SubmitJobNoWait and SubmitJobAndWait templates under the *Controls* menu allow you to submit a workflow from another workflow through using *schedulerapi*.
+
+In order to submit a workflow, you need to put the URL of that workflow in Task Variables under the key *workflowURL*. You can use whatever URL, if that URL is
+a valid workflow xml file. For example, you can add the workflow that you want to execute into *Workflow Catalog*, then retrieve its URL by indicating its Bucket ID
+and workflow ID. Moreover, the SubmitJobAndWait template will verify the status of the workflow it submits every minutes until that workflow is finished, while SubmitJobNoWait
```

Comment:
The SubmitJobAndWait will wait for the completion of the submitted workflow before proceeding further with the execution of the dependant tasks.